src/transform/mod.rs

```diff
@@ -66,11 +62,10 @@ pub fn transform_no_free_vars(
         .unwrap())
 }
 
+#[deprecated = "use [Ident::generate] instead"]
```

Comment:
That's clean, and the right thing to do once we offer stability guarantees. But as we are in version 0.2.1 < 1.0.0, hence I think we might get away with just deleting this function altogether.

Reply:
Yep, I was wondering what would be our process here. Okay, deleting it then :D